deployments/properties/aks.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Deploy to a AKS Cluster",
+    "description": "Deploy an application to a Azure Kubernetes Service Cluster",
+    "creator": "Microsoft Azure",
+    "iconName": "azure",
+    "categories": ["Deployment", "Kompose", "Helm", "Kustomize", "Kubernetes"]
```

Comment:
I noticed this later, this templates only support helm charts. Should we removed then Kompose and Kustomize as well from the categories.

Reply:
it supports all of them, but they have to make adjustments in the k8s-bake section as described in the readme for k8s-bake

Reply:
unsure if the starter workflow experience requires that to be automatic?   a bit tough as there are property name changes

Reply:
Yes, then it means user will have to modify by themselves. In that case should we add a comment suggesting user to change properties accordingly.

Reply:
Also, add  Dockerfile to the categories.

Reply:
the need to update the properties is already in the comments section at the top